.github/workflows/dest_test.yml

```diff
@@ -0,0 +1,32 @@
+name: Destination Plugin Test Workflow
+
+on:
+  pull_request:
+    paths:
+      - "plugins/destination/test/**"
```

Comment:
I think a yml file per plugin makes sense, but the issue with this type of filter (afaik) is that you can't then make it a requirement for PRs.

Reply:
what a requirement for PR mean? is it just the auto-merge ? can we skip auto-merge all together ? we don't use it that much imo and basically the rule should be just for all actions to be green?

Reply:
So I think this wont be a problem if we wont use the feature by github called "mark checks as required" because this feature is half baked and it is by default required already. it's like double require.

Reply:
Correct, it won't be a problem if we don't mark it as required. However, if a check is not required, merging is allowed even if the check fails, so it does serve a purpose. I'd prefer to enforce not allowing merging if e.g. unit tests are failing.